lib/src/base-styles/_nx-text-input.scss

```diff
@@ -14,12 +14,6 @@
   max-width: 100%;
 
   &, &.pristine, &.pristine.valid {
-    &:hover {
-      .nx-text-input__box {
-        border-color: var(--nx-color-form-element-border);
```

Comment:
Hover styling seems broken; pretty sure this removal is the reason why.

Reply:
> Hover styling seems broken; pretty sure this removal is the reason why.

Thanks, yeah I bought it back and changed it from to `--nx-color-form-element-border` to `grey-05` in this commit: 867bfa96276d741db73a1dc17d89e8ba58d04042